packages/react-components/src/templates/SharedResearchOutput.tsx

```diff
@@ -119,6 +119,9 @@ const SharedResearchOutput: React.FC<SharedResearchOutputProps> = ({
             </div>
           </Card>
         )}
+        {/* TODO:  {!isGrantDocument && (
+          <RelatedResearch relatedResearch={props.relatedResearch} />
+        )} */}
```

Comment:
was intentional to leave this commented out? I believe this is part of @oratiu task, right?